client/views/msg_action.tpl

```diff
@@ -0,0 +1,11 @@
+{{#each messages}}
+<div class="msg {{type}} {{#if self}}self{{/if}}">
+	<span class="time">
+		{{tz time}}
+	</span>
+	<span class="from"></span>
+	<span class="text">
+		<a href="#" class="user" style="color: #{{stringcolor from}}">{{mode}}{{from}}</a> {{formattedAction}} {{{parse text}}}
+	</span>
+</div>
+{{/each}}
```

Comment:
What do you think we should do for the generated templates here (I'm talking specifically here, not long term solution for handling assets)? Adding the `grunt`ed files once this is ready to be merged?